earth_enterprise/src/fusion/rasterfuse/TmeshGenerator.cpp

```diff
@@ -74,14 +77,17 @@ TmeshGenerator::Generate(float *srcSamples,
   elev.resize(SizeX * SizeY);
 
   // Error thresholds for decimation - set these to 0 to skip decimation
-  // jdjohnso - These numbers came from empirical testing?
-  // jdjohnso - The allowable error gets smaller as the resolution increases.
-  // jdjohnso - The allowable error on border tiles is 1/10 that of interior
+  // Question - Did these numbers come from empirical testing?
+  // The allowable error gets smaller as the resolution increases.
+  // The allowable error on border tiles is 1/10 that of interior
   // tiles. This is because we really want the border tiles to match. If not
   // we get rips between mesh tiles in the client.
-  int error_level = tmeshAddr.level+1;
-  double error = 0.009 * pow(2, 11-error_level);
-  double borderError = error / 10.0;
+  //int error_level = tmeshAddr.level+1;
+  // Disable decimation for spike evaluation:
+
+  double threshold_modifier = pow(2, 11 - (tmeshAddr.level + 1));
+  double error = decimation_threshold * threshold_modifier;
+  double borderError = decimate ? error / 10.0 : 0.0;
```

Comment:
What is the purpose of the `decimate` boolean here?  If we set the threshold to 0 the borderError will come out to 0 as well, so I'm not sure what it's doing for us.

Reply:
the decimate boolean is the one that comes from PacketLevelConfig in autoingest